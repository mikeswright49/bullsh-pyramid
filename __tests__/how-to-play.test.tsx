import HowToPlay from '../pages/about/how-to-play';
import { render } from '@testing-library/react';
import React from 'react';

describe('<Unit Test> - How to play', () => {
    it('should be able to render the component', () => {
        expect(render(<HowToPlay />).container).toMatchSnapshot();
    });
});
