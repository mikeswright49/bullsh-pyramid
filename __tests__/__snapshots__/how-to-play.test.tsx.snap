// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Unit Test> - How to play should be able to render the component 1`] = `
<div>
  <h1>
    How to play page
  </h1>
</div>
`;
